### PR TITLE
remove extra level check of glog

### PR DESCRIPTION
**Pull request by CaoShuFeng**
**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go [hunk 1]
```diff
@@ -76,7 +76,7 @@ func calculateBalancedResourceAllocation(pod *v1.Pod, podRequests *schedulercach
 	if glog.V(10) {
 		// We explicitly don't do glog.V(10).Infof() to avoid computing all the parameters if this is
 		// not logged. There is visible performance gain from it.
-		glog.V(10).Infof(
+		glog.Infof(
```
Comment (dixudx) [inline plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: Update the comment as well.

File: plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go [hunk 2]
```diff
@@ -231,7 +231,7 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 		if glog.V(10) {
 			// We explicitly don't do glog.V(10).Infof() to avoid computing all the parameters if this is
 			// not logged. There is visible performance gain from it.
-			glog.V(10).Infof("%v -> %v: InterPodAffinityPriority, Score: (%d)", pod.Name, node.Name, int(fScore))
+			glog.Infof("%v -> %v: InterPodAffinityPriority, Score: (%d)", pod.Name, node.Name, int(fScore))
```
Comment (dixudx) [inline plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go]: Ditto

File: plugin/pkg/scheduler/algorithm/priorities/least_requested.go [hunk 3]
```diff
@@ -75,7 +75,7 @@ func calculateUnusedPriority(pod *v1.Pod, podRequests *schedulercache.Resource,
 	if glog.V(10) {
 		// We explicitly don't do glog.V(10).Infof() to avoid computing all the parameters if this is
 		// not logged. There is visible performance gain from it.
-		glog.V(10).Infof(
+		glog.Infof(
```
Comment (dixudx) [inline plugin/pkg/scheduler/algorithm/priorities/least_requested.go]: Ditto.

And same as below.

File: plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go (hunk 1 above)
Comment (CaoShuFeng) [reply plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: Why the comment is wrong? ...

Comment (dixudx) [reply plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: `We explicitly don't do glog.V(10).Infof()`

The comment seems useless.

Comment (CaoShuFeng) [reply plugin/pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go]: I am also OK to remove all these comments.
Let's wait for someone else's suggestion.

**Comment by timothysc** [comment]
Nak, these were updated on purpose otherwise the can spam the logs b/c they are in the hotloop path.

**Comment by timothysc** [comment]
So I looked at the surrounding code and I don't know who added the if glog bit, but that seems odd, if you want to fix something, remove that.

**Comment by CaoShuFeng** [comment]
> So I looked at the surrounding code and I don't know who added the if glog bit, but that seems odd, if you want to fix something, remove that.

Hi, @timothysc this change just follows the upstream document:
https://github.com/golang/glog/blob/23def4e6c14b4da8ac2ed8007337bc5eb5007998/glog.go#L988
From:
```
if glog.V(10) { glog.V(10).Info("log this") }  // V(10) is called twice
```
to
```
if glog.V(10) { glog.Info("log this") }   // V(10) is called once, the way recommend by upstream
```

File: plugin/pkg/scheduler/algorithm/priorities/least_requested.go (hunk 3 above)
Comment (dixudx) [reply plugin/pkg/scheduler/algorithm/priorities/least_requested.go]: Remove the comment as well.

Comment (CaoShuFeng) [reply plugin/pkg/scheduler/algorithm/priorities/least_requested.go]: Done.

**Comment by timothysc** [comment]
recommended by who?  

``` glog.V(10).Info("log this")``` was the original change.

**Comment by CaoShuFeng** [comment]
https://github.com/kubernetes/kubernetes/pull/28836/files#r70439054

I wrote a benchmark test, the wrapper does have sort of better performance:
```
package main

import "testing"
import "github.com/golang/glog"
import "flag"

var intV int = 23
var stringV string = "golang"
var boolV bool = true

func init() {
        flag.Parse()
        flag.Lookup("v").Value.Set("1")
}

func BenchmarkForm1(b *testing.B) {
        for n:= 0; n < b.N; n++ {
                glog.V(10).Infof("string %s, bool %t, int %d", stringV, boolV, intV)
        }
}

func BenchmarkForm2(b *testing.B) {
        for n:=0; n < b.N; n++ {
                if glog.V(10) {
                        glog.Infof("string %s, bool %t, int %d", stringV, boolV, intV)
                }
        }
}
```
result:
```
$ go test -bench="."
goos: linux
goarch: amd64
BenchmarkForm1-8   	20000000	        69.9 ns/op
BenchmarkForm2-8   	300000000	         4.91 ns/op
PASS
ok  	_/home/fujitsu/glog_test	3.453s
```

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**